openmc/settings.py

```diff
@@ -125,6 +126,8 @@ class SettingsFile(object):
         Coordinates of the lower-left point of the UFS mesh
     ufs_upper_right : tuple or list
         Coordinates of the upper-right point of the UFS mesh
+    resonance_scattering : ResonanceScattering or iterable thereof
```

Comment:
As I understand it, the big three U-238 resonances account for most of the differences between the resonance scattering models.  Because of that, I expect a single `resonance_scattering` element will be the most common use case.  That's my use case, anyway.